### PR TITLE
Implement optout option in user data ussd line

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

File: test/ussd_popi_user_data.test.js [hunk 1]
```diff
@@ -689,10 +689,7 @@ describe('app', function() {
                                 ].join('\n')
                             })
                             .check(function(api) {
-                                // TODO: check that actually has been removed
```
Comment (rudigiesler) [inline test/ussd_popi_user_data.test.js]: Maybe here we should check that we have actually deleted the `operator` and `msisdn` fields off of the contact's answers?

**Review by rudigiesler — COMMENTED** [review]
Just one comment from me :)

**Review by rudigiesler — COMMENTED** [review]
Looks good to me.

I'm just wondering if we might need to tell the hub as well? So that it can be posted to Jembi, and so that we can cancel all subscriptions?

**Comment by KaitCrawford** [comment]
Oh right, I keep forgetting that the hub doesn't subscribe to the optout hook on the identity store. I'll make this go through the hub then rather.

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**